src/app/backend/dashboard.go

```diff
@@ -76,14 +77,17 @@ func main() {
 
 	log.Printf("Successful initial request to the apiserver, version: %s", versionInfo.String())
 
-	heapsterRESTClient, err := heapster.CreateHeapsterRESTClient(*argHeapsterHost,
-		apiserverClient)
+	// Init integrations
+	integrationManager := integration.NewIntegrationManager(clientManager)
+	err = integrationManager.Metric().
+		ConfigureHeapster(*argHeapsterHost).
+		Enable(integrationapi.HeapsterIntegrationID)
```

Comment:
So, what is the intended workflow if a user wants to use Prometheus instead of Heapster. Or maybe has already Prometheus installed and then deploys dashboard?